src/Student7/Ex3/Main.js

```diff
@@ -1,10 +1,17 @@
 import React from 'react'
+import {Tweets} from './Tweets'
 import './Main.css'
 
 export const Main = () => {
   return (
-    <div>
-        Hello I'm student id: 3.
+    <div className="Main">
```

Comment:
Let's create a `Tweet` React component.  The `Tweets` component has three `Tweet` components.